portal/src/assets/style/_mobile.scss

```diff
@@ -825,6 +832,62 @@ hr {
       color: white;
     }
   }
+
+  .asset-list-page {
+    .filter-form {
+      flex-direction: column;
+
+      &> div {
+        margin-left: 1rem;
+        margin-right: 1rem;
+        &> label {
+          &> span {
+            display: block;
+            margin-bottom: 4px;
+            font-size: 0.9rem;
+            color: $gray-700;
+          }
+        }
+      }
+
+      .query-actions {
+        button {
+          font-size: 0.9rem;
+        }
+      }
+    }
+
+    .block-section__content {
+      padding: 0 1rem 1rem 1rem;
+    }
+
+    .token-table {
+      .token-name {
+        align-items: center;
+
+        .icon {
+          width: 24px;
+          height: 24px;
+        }
+
+        .info {
+          font-size: 0.9rem;
+        }
+      }
+    }
+
+    hr {
+      width: initial;
+      margin-left: 1rem;
+      margin-right: 1rem;
+      height: 1px;
+    }
+  }
+
+  .mobile-margin-x {
```

Comment:
Please give a better name , what is X?

Reply:
x-axis, from left to right. But i will give it a  better name